app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,73 @@
+class PostsController < ApplicationController
```

Comment:
Uses `before_action` to restrict access to the `:new` and `:create` methods (uses ApplicationController helper method) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/before_action_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))